src/test/java/pages/BasePage.java

```diff
@@ -72,5 +74,9 @@ public static void selectByValue(WebElement element, String value) {
         countrySelector.selectByValue(value);
     }
 
-
+    public static void waitElementHasClass(WebElement element, String elementClass) throws InterruptedException {
```

Comment:
Added this method because for some reason the site has a slight delay between moving the course in the carousel and assigning the swiper-slide-next class to it. I tried different solutions, this turned out to be the most workable.